modules/nodes-descheduler-configuring-profiles.adoc

```diff
@@ -35,30 +35,33 @@ spec:
   logLevel: Normal
   managementState: Managed
   operatorLogLevel: Normal
+  mode: Predictive                                     <1>
```

Comment:
maybe mention if it's mandatory or optional. and default mode s predictive

Reply:
@RoniKishner Is this mandatory or optional?

Reply:
Optional, only 2 values are possible ["Automatic", "Predictive"], as geetika mentioned default is "Predictive"